Lesson26/Gokhan/index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <title>Image Gallery</title>
+  <link rel="stylesheet" type="text/css" href="./styles.css">
+</head>
+
+<body>
+
+  <h3>Image Gallery</h3>
+
+  <br>
+
+  <label for="image-count">Image Count</label>
+
+  <select id="image-count">
+    <option value="5">5</option>
+    <option value="10">10</option>
+    <option value="15">15</option>
+    <option value="20">20</option>
+    <option value="25">25</option>
+  </select>
+
+  <button id="fetch-button" class="button">Fetch Images</button>
+
+  <div id="image-container"></div>
+
+  <br><br>
+  <button id="reset-button" type="reset">Clear Images</button>
```

Comment:
It would be better to move this button above `<div id="image-container"></div>` so it's visible on the page even when there are plenty of images